qemu/tests/cfg/balloon_service.cfg

```diff
@@ -1,32 +1,39 @@
 - balloon_service: install setup image_copy unattended_install.cdrom
     virt_test_type = qemu
     type = balloon_service
-    only Windows
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
     no Host_RHEL.m5, Host_RHEL.m6
     monitor_type = "qmp"
     monitors = qmp1
     balloon = balloon0
     balloon_dev_devid = balloon0
     balloon_dev_add_bus = yes
-    driver_name = "balloon"
-    cdroms = "cd1 winutils virtio"
-    drive_index_virtio = 3
-    cdrom_virtio = isos/windows/virtio-win-1.8.0-4.iso
-    # balloon service related cmd, which will be covered by different
-    # guest's paths in guest-os cfg.
-    install_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -i"
-    uninstall_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -u"
-    status_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe status"
-    run_balloon_service = "%s:\Balloon\GUEST_OS\amd64\blnsvr.exe -r"
+    Windows:
+        driver_name = "balloon"
+        cdroms = "cd1 winutils virtio"
+        cdrom_virtio = isos/windows/virtio-win-1.8.0-4.iso
```

Comment:
This is just specify that param cdrom_virtio is a must for this case, about the version, it will be covered by internal one.